vault/resource_gcp_secret_impersonated_account.go

```diff
@@ -68,6 +68,12 @@ func gcpSecretImpersonatedAccountResource() *schema.Resource {
 				Computed:    true,
 				Description: "Project of the GCP Service Account managed by this impersonated account",
 			},
+			consts.FieldTTL: {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    false,
```

Comment:
nit: redundant since false is the default

Reply:
Removed the line :)